Gambot.IO.Slack/SlackMessage.cs

```diff
@@ -11,19 +14,22 @@ public class SlackMessage : IMessage
         public string Where { get; protected set; }
         public bool Action { get; protected set; }
 
-        public SlackMessage(NewMessage raw)
+        public SlackMessage(NewMessage raw, IDictionary<string, User> userLookup)
         {
-            Who = raw.user;
+            Who = raw.subtype == "bot_message"
+                ? "slackbot" // probably
```

Comment:
![mj-lol](https://emilh.net/img/mj-lol.gif)

Reply:
this library doesn't expose bot_id, we just need Who to not be null